src/styles/card.css

```
@@ -157,13 +157,14 @@
   justify-content: center;
-  mask-image: linear-gradient(rgba(0, 0, 0, 0.1) 0%, white 20px);
+  mask-image: linear-gradient(white 0, white 0.2rem);
   perspective: 1000px;
-  top: calc(50% - 250px);
+  top: calc(50% - 242px);
   left: 0;
   right: 0;
   height: 100%;
-  min-height: 1500px;
+  margin-top: -11rem;
+  height: 100%;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)